src/ApiService/ApiService/onefuzzlib/Storage.cs

```diff
@@ -0,0 +1,79 @@
+using System.Collections.Generic;
+using System;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Storage;
+using Azure.Core;
+using Microsoft.Extensions.Logging;
+using System.Text.Json;
+
+namespace Microsoft.OneFuzz.Service;
+
+public interface IStorage {
```

Comment:
you can use StorageProvider.cs for this

Reply:
I was trying to do a direct mapping of this file: https://github.com/microsoft/onefuzz/blob/main/src/api-service/__app__/onefuzzlib/azure/storage.py

Reply:
I can move it to StorageProvider.cs if you still think it makes more sense to be there

Reply:
That's ok you can keep  it this way. We will probably need a pass to unify both